setup.py

```diff
@@ -79,6 +82,10 @@
             'twine',
             'wheel',
         ],
+        'standalone_executor': [
```

Comment:
@jberci, it turns out `extras_require` can't be used to specify an arbitrary set of dependencies (like standalone executor's dependencies) but [can only be used for declaring optional features with their own dependencies](
http://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies).
That means the base `install_requires` will always be installed along with them, which makes this useless in our case.

I propose another solution then. Create a `requirements.txt` file inside `resolwe/flow/executors` with the following contents:
```
# Python requirements for running Resolwe's standalone executor
redis
six
```

To include this in the package, add it to the `package_data` in `setup.py`.

This file can then be used to prepare the standalone executor's Python environment in a production setting.